src/core/server/saved_objects/routes/copy.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema } from '@osd/config-schema';
+import { IRouter } from '../../http';
+import { SavedObjectConfig } from '../saved_objects_config';
+import { exportSavedObjectsToStream } from '../export';
+import { importSavedObjectsFromStream } from '../import';
+
+export const registerCopyRoute = (router: IRouter, config: SavedObjectConfig) => {
+  const { maxImportExportSize } = config;
+
+  router.post(
+    {
+      path: '/_copy',
```

Comment:
1. Is there any plan to make this API generic? For example today we have only workspace. So this copy API will copy saved objects to workspace. But say in future we have entity A, entity B something like that. So my question is based on parameter are we planning to support cop saved object to entity A, copy saved objects to entity B? If not, I would suggest to make it more self explanatory something like `_copyToWorkspace`? 

2. Is there plan to add api to delete saved objects from workspace?

3. Is there impact on saved object client wrapper? As you are using exiting import export, so I am guessing no. Can you confirm that?

4. Also how the permission plays role for this API? Does any user can call this API and copy saved objects to any workspace?
Thanks!

Reply:
Thanks for your comment!
1. This new API is only used to copy to workspace, so I think it makes sense that we change the name to `_copyToWorkspace`.
2. We'll add a new function `deleteByWorkspace` to the saved objects client, it will only be used in the server side, so no api needs to add.
3. No, for this API, nothing change in the saved object client wrapper.
4. The permission control performs on the `WorkspaceSavedObjectsClientWrapper` which will be added in the following PR, for this API, when calling `savedObjectClient.bulkGet()` and `savedObjectsClient.bulkCreate()`, we'll check if the user has required permissions based on the `permissions` field and `workspaces` field of the saved objects.

Reply:
So if I understand correctly this API will copy a saved object to destination workspace, right? And Id and all the fields will remain same of the same object in both workspaces. Is that right? Will it create a new document or update one field (say workspace) for existing saved object?

Can you please attach the video of demonstration?

Reply:
No, we create a new ID for the duplicated saved object in the target workspace,  correspondingly a new document is created in the `.kibana` index. I've made a video about the duplicate UI, please take a look.

Reply:
Thanks for adding video.